joomla/joomla-cms
# [4.0] Add a sample workflow to blog sample data

### Summary of Changes
A sample workflow is added to sample data. 

<img width="901" alt="workflow-sample3" src="https://user-images.githubusercontent.com/1035262/95874388-2188a180-0d71-11eb-94d9-0540e3b47961.PNG">



### Testing Instructions
You need a fresh installation with a clean database.

Tip: you can copy the empty database (phpmyadmin) in case you need again an empty database.

Apply the patch, go to the main dashboard and install blog samole data.

Refresh the Screen (because the workflow must be loaded into the menu). Then you should have a new workflow in your content area: 

<img width="548" alt="workflow-sample" src="https://user-images.githubusercontent.com/1035262/84564959-0c214d80-ad66-11ea-8c31-213bf7495818.PNG">
